### ISSUE TITLE
NOT WORKING

### ISSUE THREAD

**Issue by DanielLivingston32** [atlonxp/recursive-goIndex-downloader#11]
It is not able to crawl the following website
https://webdev.1csalgo.workers.dev/0:/Academind%20Courses/React%20-%20The%20Complete%20Guide%20(incl%20Hooks,%20React%20Router,%20Redux)/

**Comment by vinc3nati** [comment]
Hey I'm trying to open the colab notebook but it asks for permission . If you can send me your colab copy of this code of **version 2**  I can help you with that.

**Comment by rushi3691** [comment]
> It is not able to crawl the following website
> https://webdev.1csalgo.workers.dev/0:/Academind%20Courses/React%20-%20The%20Complete%20Guide%20(incl%20Hooks,%20React%20Router,%20Redux)/

it requires username and password.
write it to me and i'll check the pattern

**Comment by DanielLivingston32** [comment]
> > It is not able to crawl the following website
> > https://webdev.1csalgo.workers.dev/0:/Academind%20Courses/React%20-%20The%20Complete%20Guide%20(incl%20Hooks,%20React%20Router,%20Redux)/
> 
> it requires username and password.
> write it to me and i'll check the pattern

https://csalgoweb.telegramm.workers.dev/
Username & Password: VanGogh

**Comment by tipitopzui** [comment]
Did it worked? @DanielLivingston32

**Comment by codechef-karunya** [comment]
Nope. It doesn't work for sites with username and password.

**Comment by DanielLivingston32** [comment]
Nope. It doesn't work for sites that require username and password.
